src/App.js

```diff
@@ -1,30 +1,31 @@
 import React from 'react';
-
 import './App.scss';
-
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import { PostList } from './components/PostList';
 
-const App = () => (
+const preparedPost = posts.map(post => ({
```

Comment:
post - singular
posts - plural

```suggestion
const preparedPosts = posts.map(post => ({
```